notebooks/simple_xgboost_example.ipynb

```diff
@@ -0,0 +1,714 @@
+   "source": [
+    "## Setup\n",
+    "\n",
+    "Before running this notebook, please check whether NVIDIA driver, Docker and NVIDIA CUDA Toolkit. `nvidia-smi` command should run successfully as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!nvidia-smi"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install XGBoost and Sklearn\n",
+    "\n",
+    "We'd need to install XGBoost and SKlearn using the following pip3 commands inside the container as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install sklearn first\n",
+    "!pip3 install -U scikit-learn\n",
+    "\n",
+    "# Then install XGBoost\n",
+    "!pip3 install xgboost"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## XGBoost model\n",
+    "\n",
+    "If you have a pre-trained xgboost model, save it as `xgboost.model` and skip this step. We'll train a XGBoost model on random data in this section "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import required libraries\n",
+    "import numpy\n",
+    "from numpy import loadtxt\n",
+    "from xgboost import XGBClassifier\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import accuracy_score\n",
+    "\n",
+    "import os\n",
+    "import signal\n",
+    "import subprocess"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Generate dummy data to perform binary classification\n",
+    "seed = 7\n",
+    "features = 9 # number of sample features\n",
+    "samples = 10000 # number of samples\n",
+    "X = numpy.random.rand(samples, features).astype('float32')\n",
+    "Y = numpy.random.randint(2, size=samples)\n",
+    "\n",
+    "test_size = 0.33\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X, Y, test_size=test_size, random_state=7)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = XGBClassifier()\n",
+    "model.fit(X_train, y_train)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y_pred = model.predict(X_test)\n",
+    "predictions = [round(value) for value in y_pred]\n",
+    "accuracy = accuracy_score(y_test, predictions)\n",
+    "print(\"Test Accuracy: %.2f%%\" % (accuracy * 100.0))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Export and load XGBoost model in Triton Inference Server\n",
+    "\n",
+    "For deploying the trained XGBoost model in Triton Inference Server, follow the steps below:\n",
```

Comment:
Just to help underscore that the name is arbitrary, why don't we name this model `xgboost_example` and update the directory name accordingly?